docs/UserGuide.md

```diff
@@ -75,114 +85,112 @@ Shows a message explaning how to access the help page.
 
-Format: `delete NRIC`
+Shows a list of all patients in CLInic.
 
-* Deletes the person with specified `NRIC`.
-* The NRIC **must exist within database**.
-
-Examples:
-* `list` followed by `delete S1234567A` deletes the person with NRIC S1234567A in the address book.
-* `find Betsy` followed by `delete S1234567A` deletes the person with NRIC S1234567A in the results of the `find` command.
+Format: `list`
 
-### Adding an Appointment: `addApp`
+### Adding an Appointment: `addAppt`
 
-Adds an appointment to the address book.
+Adds an appointment to the CLInic.
 
-Format: `addApp i/NRIC d/DATE from/START_TIME to/END_TIME t/APPOINTMENT_TYPE [note/NOTE]`
+Format: `addAppt i/NRIC d/DATE from/START_TIME to/END_TIME t/APPOINTMENT_TYPE [note/NOTE]`
 
-* Adds an appointment for the person with specified `NRIC`, on `DATE` from `START_TIME` to `END_TIME`
-* Person with this NRIC **must exist within database**.
+* Adds an appointment for the patient with specified `NRIC`, on `DATE` from `START_TIME` to `END_TIME`
+* Patient with this NRIC **must exist within database**.
 * Details of `APPOINTMENT_TYPE` and `NOTE` will be captured for reference
 * `note/` is an optional field
 
 Examples:
-* `addApp i/ T0123456A d/ 2024-02-20 from/ 11:00 to/ 11:30 t/ Medical Check-up note/ Routine check-in`
-* `addApp i/ S1234567A d/ 2024-02-20 from/ 15:00 to/ 15:30 t/ Blood Test note/ Follow-up from last consultation`
+* `addAppt i/ T0123456A d/ 2024-02-20 from/ 11:00 to/ 11:30 t/ Medical Check-up note/ Routine check-in`
+* `addAppt i/ S1234567A d/ 2024-02-20 from/ 15:00 to/ 15:30 t/ Blood Test note/ Follow-up from last consultation`
 
-### Cancelling an Appointment: `cancelApp`
+### Cancelling an Appointment: `cancelAppt`
 
-Cancels an appointment from the address book.
+Cancels an appointment from CLInic.
 
-Format: `cancelApp i/NRIC d/DATE from/START_TIME to/END_TIME`
+Format: `cancelAppt i/NRIC d/DATE from/START_TIME to/END_TIME`
 
-* Cancels an appointment for the person with specified `NRIC`, on `DATE` from `START_TIME` to `END_TIME`
+* Cancels an appointment for the patient with specified `NRIC`, on `DATE` from `START_TIME` to `END_TIME`
 * Appointment with the following details **must exist within database**.
 
 Examples:
-* `cancelApp i/ S8743880A d/ 2024-02-20 from/ 11:00 to/ 11:30`
+* `cancelAppt i/ S8743880A d/ 2024-02-20 from/ 11:00 to/ 11:30`
 
 ### Finding appointments: `findAppt`
 
 Finds appointments based on the given parameters.
 
 Format: `findAppt [i/NRIC] [d/DATE] [from/START_TIME]`
 
-* Filters an appointment with specific `NRIC`, `DATE` or `START_TIME`
+* Filters an appointment with specific `NRIC`, `DATE` or `START_TIME` (any combination of the 3)
 * If invalid parameters, error detailing what went wrong will be displayed.
 * For argument concerning TIME, all appointments that start at the given time and later than that are returned.
 * Fetching for TIME without DATE will return all appointments whose start time >= that time on any date.
```

Comment:
Not too sure if mathematical symbols ">=" here will be ideal. Maybe this can use "Examples" similar to what the other commands have.